spec/datadog/appsec/extensions_spec.rb

```diff
@@ -59,12 +60,12 @@
 
       describe '#enabled' do
         subject(:enabled) { settings.enabled }
-        it { is_expected.to eq(true) }
+        it { is_expected.to eq(false) }
       end
 
       describe '#enabled=' do
-        subject(:enabled_) { settings.enabled = false }
-        it { expect { enabled_ }.to change { settings.enabled }.from(true).to(false) }
+        subject(:enabled_) { settings.enabled = true }
+        it { expect { enabled_ }.to change { settings.enabled }.from(false).to(true) }
       end
```

Comment:
Is this repeated from `settings_spec`? I see a lot of things that are the same between both. (Do we still keep both copies around, or could we perhaps drop one of them?)